esx_service/utils/auth_api.py

```diff
@@ -28,6 +28,7 @@
 
 # regex for valid tenant name
 VALID_TENANT_NAME_REGEXP = "[a-zA-Z0-9_][a-zA-Z0-9_.-]*"
+VALID_TENANT_NAMES = 'rename of vm-groups other than _DEFAULT'
```

Comment:
In the message the highlited parts are fixed for the error string define for TENANT_NAME_INVALID
## Vm-group name _DEFAULT is invalid, only _rename of vm-groups other than DEFAULT ## is allowed